### ISSUE TITLE
Master issue revamped subtraction

### ISSUE THREAD

**Issue by jan-petr** [ExploreASL/ExploreASL#1551]
### Description

Issues #1096 and #997 both edit the DEBBIE concatenation and how PWI subtraction is calculated. To finish these two tasks, there is a follow-up issues #1543 that can be only completed once these two branches are merged. 

There will be a provisional branch related to this issue and all issue mentioned here will be merged to this branch.
Note that steps have to be done sequentially, but issues within the same step can be done in parallel. 
- [x] Step 1 - Branch #1551 is based on `develop` after #1563 and #1577 are merged.
- [x] Step 2 - Branch #1096 will be merged into it and their conflicts resolved.
- [x] Step 3 - Branch #997  will be merged into it and their conflicts resolved.
- [x] Step 3 - UnitTest
- [x] Step 3 - FlavorTest
- [x] Step 4 - TestDataSets #1581 
- [x] Step 4 - #1585 
- [x] Step 4 - #1211 
- [x] Step 4 - #1601 
- [x] Step 5 - #1539 can be done anytime
- [x] Step 5 -  #1549    
- [x] Step 5 - #1603 is first part of the follow-up.
- [x] Step 6 - after merging all, rebase the branch with merged #1584, #1587, #1599,  #1596, #1590, #1610, #1597, #1535, #1613, #1081, #1071   
- [x] Step 7 - #1543 cannot be done in parallel with other issues.
- [x] Step 8 - Run and update values for TestDataSet
- [x] TestDataSet - TC ASL and TC registration doesn't work.
- [x] Step 9 - This branch will be then finally merged to develop. No more testing of it will be needed, as final testing can be done on #1543 , but we might do a last formal review of the code changes of this branch versus `develop`.

- [x] Expand the functionality of xASL_delete to also delete .mat & .json sidecars by default (integrating xASL_adm_DeleteFilePair), and apply this throughout the pipeline. JAN: Perhaps in a new issue later?
- [x] Add the scaling list parameter into the documentation. In documentation add SessionMergingScaling and change sessionMergingList to SessionMergingList. JAN will do this before merging.
- [x] T1b should be 1.7306679, T1t should be 1.33 HENK: why? We need reasoning for this. Note that "this is what FME/DEBBIE wants" cannot be the only motivation. JAN All previous DEBBIE code used that so we wanted to have that for checking that the behavior is the same as it was. Jan is fine with keeping 1650 ms and 1240 ms for all. HENK: We should (I think we already have this) have T1b as option, which would solve the comparison issue. As a default T1b: I would argue we use something that has most consensus (e.g., white paper or a newer gray paper). I find it tricky to use what Amnah uses — even if it is better — because ExploreASL should be comparable with other white/gray paper-based pipeline. Ideally, the new gray paper by Joe Woods/Michael already recommends the same values as Amnah uses, then we have a good reason to take this new default. We can of course print this parameter extra with its source, doesn't hurt to make the user aware of the defaults we use and why, especially when there are multiple standards (if a lot of people use the DEBBIE sequence then this also becomes a de facto standard)JAN Lets move this to a new DEBBIE issue.
- [x] Manage how to deal with SD & SNR currently saved in xASL_wrp_Resample. xASL_wrp_Resample is for:PWI4D that will later be used for quantification (after potential merging/manual adaptations). Other reference images for PVC, visual QC, etc. For this, we want all available SNR we can get, similar to the temporary PWI created in xASL_wrp_RegisterASL. This is why we want to do the same weighting here as we did in xASL_wrp_RegisterASL (probably by a single weighting function). Historically, we created SD, SNR, and other QC stuff in xASL_wrp_Resample. The question is where we keep this, or if we move this to after quantification. Because we probably want this to be AFTER the merging/manual removal right? (please also think about Mervin merging two single-PLD, one at the start and one at the end of the scan protocol, not only think about a single exotic DEBBIE sequence, but think generalized ;) -> So we could move this SD/SNR part to xASL_wrp_VisualQC, that would make most sense to me. I suggest we do this move in this issue, and do the QC improvement later. JAN Agree - should we do this in a new issue then?

### Release notes

Nothing. Add to #1096 and #997.
